src/fancy-tools/wwwcRegionTool.js

```diff
@@ -147,63 +73,135 @@ export default class extends baseTool {
    * @param  {} evt
    */
   _setHandlesAndUpdate (evt) {
-    const eventData = evt.detail;
-    const element = eventData.element;
-
-    this.handles.end = eventData.currentPoints.image;
+    const element = evt.detail.element;
+    const image = evt.detail.currentPoints.image;
 
+    this.handles.end = image;
     external.cornerstone.updateImage(element);
   }
 
   /**
-  * Event handler for MOUSE_UP during handle drag event loop.
-  *
-  * @param {Object} evt - The event.
-  */
-  _mouseUpCallback (evt) {
+   * Event handler for MOUSE_UP during handle drag event loop.
+   *
+   * @param {Object} evt - The event.
+   */
+  _applyStrategy (evt) {
     if (isEmptyObject(this.handles.start) || isEmptyObject(this.handles.end)) {
       return;
     }
 
-    const eventData = evt.detail;
-    const element = eventData.element;
-
-    eventData.handles = this.handles;
-    this.applyActiveStrategy(evt);
+    evt.detail.handles = this.handles;
+    applyWWWCRegion(evt, this.configuration);
     this._resetHandles();
-    this._deactivateModify(element);
   }
 
+  /**
+   * Sets the start and end handle points to empty objects
+   *
+   */
   _resetHandles () {
     this.handles = {
       start: {},
       end: {}
     };
   }
+}
 
-  /**
-  * Adds modify loop event listeners.
-  *
-  * @private
-  * @param {Object} element - The viewport element to add event listeners to.
-  * @modifies {element}
-  */
-  _activateModify (element) {
-    element.addEventListener(EVENTS.MOUSE_UP, this._mouseUpCallback);
-    element.addEventListener(EVENTS.MOUSE_CLICK, this._mouseUpCallback);
-    element.addEventListener(EVENTS.MOUSE_MOVE, this._mouseMoveCallback);
+/**
```

Comment:
I agree helpers look nicer at the bottom, so you see the class upon opening the file. I've wondered though (and I'm guilty of the same), if we aren't reusing these somewhere, should they not just be class methods?

Reply:
I could see them being private class methods. I wasn't sure if they were generic enough to be pulled into a shared module as potentially reusable code though?

Reply:
I think we should leave it as you have it now for this PR. Nearer the end we can see if this functionality should be pulled out or if we should make it private.